src/modules/dashboard/components/ActionsPage/StakeRequiredBanner/StakeRequiredBanner.css

```diff
@@ -11,7 +11,41 @@
   padding: 9px 0;
 }
 
+.share {
+  position: absolute;
+  top: 11px;
+  right: 20px;
+}
+
 .share button {
-  margin-left: 4px;
-  font-size: var(--size-small);
+  padding-right: 6px;
+  padding-left: 10px;
+  border: 0px;
+  color: var(--action-secondary);
+  opacity: 1;
+
+  &:hover {
+    border: 1px solid var(--action-secondary);
+    background-color: rgba(255, 255, 255);
```

Comment:
Not limited to your PR but I'm wondering why there is no var for rgba(255, 255, 255) i.e. white???